pkgs/applications/virtualization/docker/default.nix

```diff
@@ -282,8 +282,8 @@ rec {
     tiniHash = "sha256-ZDKu/8yE5G0RYFJdhgmCdN3obJNyRWv6K/Gd17zc1sI=";
   };
 
-  docker_24 = callPackage dockerGen rec {
-    version = "24.0.5";
+  docker_25 = callPackage dockerGen rec {
+    version = "25.0.3";
     cliRev = "v${version}";
     cliHash = "sha256-u1quVGTx/p8BDyRn33vYyyuE5BOhWMnGQ5uVX0PZ5mg=";
```

Comment:
No hash update ?!

Reply:
It also seemed wrong, but the version did change when doing docker --version and it built successfully

Reply:
@aikooo7 yeah because the version is set by the build itself. If you change a random character from the hashes and re-run the build, you'll see a failure (and the new hashes as well).